db_stress_tool/db_stress_tool.cc

```diff
@@ -84,6 +85,16 @@ int db_stress_tool(int argc, char** argv) {
     raw_env = fault_env_guard.get();
   }
 #endif
+
+  if (FLAGS_env_fault_injection) {
```

Comment:
Can we combine this block with the one above and use ```FaultInjectionTestFS```, which offers the same functionality?